packages/flame/lib/src/components/core/component.dart

```diff
@@ -1053,7 +1046,7 @@ class _LifecycleManager {
       oldParent?.onChildrenChanged(child, ChildrenChangeType.removed);
       child._parent = owner;
       child._mount();
-      owner.onChildrenChanged(child, ChildrenChangeType.added);
+      // owner.onChildrenChanged(child, ChildrenChangeType.added);
```

Comment:
?

Reply:
The `_mount()` method already does 
```
_parent!.onChildrenChanged(this, ChildrenChangeType.added);
```
at the end, so this call would create a second notification.

Reply:
So it should be removed then? :)